app/src/test/java/ingsw/group1/findmyphone/location/GeoPositionTest.java

```diff
@@ -0,0 +1,176 @@
+package ingsw.group1.findmyphone.location;
+
+import android.location.Location;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.fail;
+
+
+/**
+ * Test class for {@link GeoPosition}.
+ *
+ * @author Riccardo De Zen.
+ */
+@Config(sdk = 28)
+@RunWith(RobolectricTestRunner.class)
+public class GeoPositionTest {
+
+    private double exampleLatitude = 111.111d;
+    private double exampleLongitude = 99.999d;
+    private GeoPosition examplePosition;
+
+    /**
+     * Checking the default constructor passes.
+     */
+    @Before
+    public void doesConstructorPass() {
+        examplePosition = new GeoPosition(exampleLatitude, exampleLongitude);
+    }
+
+    /**
+     * Testing {@link GeoPosition#getLatitude()} returns the actual value of the latitude.
+     */
+    @Test
+    public void getLatitude_returnsActual() {
+        assertEquals(exampleLatitude, examplePosition.getLatitude());
+    }
+
+    /**
+     * Testing {@link GeoPosition#getLongitude()} returns the actual value of the longitude.
+     */
+    @Test
+    public void getLongitude_returnsActual() {
+        assertEquals(exampleLongitude, examplePosition.getLongitude());
+    }
+
+    /**
+     * Testing {@link GeoPosition#setLatitude(double)} sets a new value of the latitude.
+     */
+    @Test
+    public void setLatitude_setsCorrectly() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'setLatitude_setsCorrectly' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4950325)